src/usr/local/www/system_gateways.php

```diff
@@ -142,8 +142,9 @@ function delete_gateway_item($id) {
 	    !isset($a_gateways[$id]['disabled']) &&
 	    isset($a_gateways[$id]['isdefaultgw'])) {
 		$inet = (!is_ipaddrv4($a_gateways[$id]['gateway']) ? '-inet6' : '-inet');
+		$rgateway = $a_gateways[$id]['gateway'];
```

Comment:
Why set `$rgateway` when it's only used once in these instances? Why not just use `$a_gateways[$id]['gateway']` directly?

Reply:
done